man/systemd-cryptenroll.xml

```diff
@@ -549,6 +549,10 @@
         project='mankier'><refentrytitle>tpm2_dictionarylockout</refentrytitle><manvolnum>1</manvolnum></citerefentry>
         commands, respectively.</para>
 
+        <para>Note as well that the new pin can be provided through the NEWPIN environment variable. In case
```

Comment:
Don't document the env var here, it's only for testing

Reply:
done